package.json

```diff
@@ -522,6 +522,16 @@
                 "type": "boolean",
                 "description": "Determines whether a temporary PowerShell Extension Terminal is created for each debugging session, useful for debugging PowerShell classes and binary modules.  Overrides the user setting 'powershell.debugging.createTemporaryIntegratedConsole'.",
                 "default": false
+              },
+              "attachDotnetDebugger": {
+                "type": "boolean",
+                "description": "If specified, a C# debug session will be started and attach to the new temporary PSIC. This does nothing unless createTemporaryIntegratedConsole is also specified.",
+                "default": false
+              },
+              "dotnetDebuggerConfigName": {
+                "type": "string",
+                "description": "If you would like to use a custom coreclr attach debug launch configuration for the debug session, specify the name here, otherwise a default basic config will be used. The config MUST be a coreclr attach config, launch configs are not supported.",
```

Comment:
```suggestion
                "description": "If you would like to use a custom coreclr attach debug launch configuration for the debug session, specify the name here. Otherwise a default basic config will be used. The config must be a coreclr attach config. Launch configs are not supported.",
```